.github/workflows/dry-run-build.yml

```
@@ -0,0 +1,18 @@
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+      - name: Setup R and Bioconductor
+        uses: grimbough/bioc-actions/setup-bioc@v1
+        with:
+          bioc-version: devel
+      - name: Install dependencies
+        uses: r-lib/actions/setup-r-dependencies@v2
+      - name: Build, Install, Check
+        uses: grimbough/bioc-actions/build-install-check@v1
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)